### DISCUSSION TITLE
Map viewer

### DISCUSSION THREAD

**Discussion by AnaBibiW** [lsc-hubs/kenya-hub#12] [General]
Is it possible for the comparison of the maps to be done side by side? the line running through one map and another is not giving a clear visual comparison of two different elements

Also, some maps do not have  a legend

**Comment by pvgenuchten** [comment]
hi @AnaBibiW, please use the side-by-side tool for this

![image](https://github.com/lsc-hubs/kenya-hub/assets/299829/fb03105b-5fb4-4637-84ca-05241dfe86b6)
